src/pages/storage/StorageVolumes.tsx

```diff
@@ -210,14 +229,15 @@ const StorageVolumes: FC = () => {
 
   const snapshotsPerVolume = getSnapshotsPerVolume(volumes);
   const rows = filteredVolumes.map((volume) => {
+    const key = getVolumeKey(volume);
     const volumeType = renderVolumeType(volume);
     const contentType = renderContentType(volume);
-
-    const key = `${volume.name}-${volume.pool}-${volume.location}`;
     const snapshotCount = snapshotsPerVolume[key]?.length ?? 0;
+    const canSelect = canDeleteVolume(volume) && hasVolumeDetailPage(volume);
```

Comment:
```suggestion
    const canSelect = hasVolumeDetailPage(volume);
```
Let's allow selecting also volumes without permissions. We can rely on the `StorageVolumeBulkDelete` component to surface the missing permission.